inc/functions.php

```diff
@@ -187,3 +187,22 @@ function get_cookie_policy_url() : string {
 function get_category_label( string $category ) : string {
 	return consent_category_labels()[ $category ] ?? '';
 }
+
+/**
+ * Retrieve the always allowed categories.
+ *
+ * @return array Returns always allowed categories.
+ */
+function get_always_allowed_categories() {
+	$allowed = apply_filters( 'altis.consent.always_allow_categories', [] );
```

Comment:
Filters should ideally have a docblock above them, similar to a function's docblock but without the `@return` part.